Sources/Extensions/SwiftStdlib/BoolExtensions.swift

```diff
@@ -61,5 +61,5 @@ public extension Bool {
 		self = !self
```

Comment:
Should we deprecate this one? Now I think is in the stdlib

Reply:
Yes I think that's a good idea

Reply:
Should we deprecate both `toggle` and `toggled`?

Reply:
Yes

Reply:
The built-in `toggle()` in Swift 4.2 conflicts with this, neither can be called. 

I ran into this problem because I am currently using Swift 4.2, I temporarily removed this method locally to solve this problem.

Maybe we need to remove this method in the next release, or add a swift version check condition.